components/ui/expandable.tsx

```
@@ -0,0 +1,538 @@
+
+ExpandableCard.displayName = "ExpandableCard"
+
+// I'm telling you we just have to expand 🤌💵
+const ExpandableTrigger = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ children, ...props }, ref) => {
+  const { toggleExpand } = useExpandable()
+  return (
+    <div ref={ref} onClick={toggleExpand} className="cursor-pointer" {...props}>
+      {children}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)